Sources/Components/SelectionBoxes/Selectionbox.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+/* Selection box for selecting a single item */
+
+public protocol RadioButtonDelegate: class {
+    func radioButton(_ radioButton: RadioButton, didSelectItem item: SelectionboxItem)
```

Comment:
I think this should be RadioButtonItem instead of SelectionboxItem same for the Checkbox.

Reply:
I thought maybe use index instead of item. There's not really anything useful in the items at the moment so maybe just make it an internal class for now?

Reply:
I think you might need the item in case it has an ID for example and we'll most likely make it a protocol like we did with the other components. For now I think it would be nice the types so the radio button doesn't have to deal with Selectionbox items.